src/Schema/Events/Tap.ts

```diff
@@ -1063,3 +1063,29 @@ export interface TappedNewsSection {
   destination_screen_owner_type: ScreenOwnerType
   destination_screen_owner_id: string
 }
+
+/**
+ * A user clicks on View Work on the notification page.
+ *
+ * This schema describes events sent to Segment from [[tappedViewWork]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "tappedViewWork",
+ *    context_module : "Notifications"
```

Comment:
question: It seems we don't have `notifications` context_module. We do have `notification`, though.